open-metadata-implementation/access-services/digital-service/digital-service-api/README.md

```diff
@@ -0,0 +1,8 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+# Digital Service Open Metadata Access Service (OMAS) API
+
+The Digital Service OMAS supports a REST API to support the lifecycle of an Egeria Digital Service
+The Digital Service OMAS API provides the shared Java classes between the
```

Comment:
probably 'OMAS API module' - since this isn't really what the reader might consider a published api - it's an implementation detail and the comment is on the module structure?